### DISCUSSION TITLE
The operation couldn't be completed. (Keychain.Keychain.Error error 1.)

### DISCUSSION THREAD

**Discussion by RodrigoIbarraSanchez** [intitni/CopilotForXcode#563] [General]
I can't add the OpenAI key, today my key disappeared and when I try to add it again I have the The operation couldn't be completed. (Keychain.Keychain.Error error 1.) error. Do you have any idea why?

**Comment by intitni** [comment]
Sorry for the late reply. I am not sure what the cause is, but it seems that it can't set an item in the keychain. Please try the following:

- Open Keychain.app, search for com.intii.CopilotForXcode and delete everything you see. And try adding it again. 
- If the above doesn't work, consider turning on "Store API keys in UserDefaults" in the advanced settings.
